equipment_handler/source/adc_handler.c

```diff
@@ -0,0 +1,371 @@
+/*
+ * Copyright (C) 2015  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+/**
+ * @file    adc_handler.c
+ * @author  Vasu Gupta
+ * @date    2020-06-15
+ */
+
+#include <ex2_hal/ex2_hyperion_solar_panel_software/equipment_handler/include/adc_handler.h>
```

Comment:
Could you try again adding "ex2_hal/ex2_hyperion_solar_panel_software/equipment_handler/include" to the include paths and see if you can improve this?

Reply:
I just tried that again but no luck. Not sure why. I guess we just leave it like this for a while.